Kickstarter-iOS/Views/Cells/CommentCell.swift

```diff
@@ -25,7 +25,8 @@ internal final class CommentCell: UITableViewCell, ValueCell {
 
     _ = self
       |> baseTableViewCellStyle()
-      |> CommentCell.lens.contentView.layoutMargins .~
+      |> UITableViewCell.lens.backgroundColor .~ .white
```

Comment:
would we wanna bake this background color into the base table view style so that we didnt have to update it everywhere?